archivo/querying/query_databus.py

```diff
@@ -115,8 +115,41 @@ def get_version_val(b) -> str:
     )
 
 
+def find_previous_version(versions, target_file, target_version):
+    target_datetime = datetime.strptime(target_version, '%Y.%m.%d-%H%M%S')
+    previous_version = None
+    
+    for version_dict in versions:
+        if target_file in version_dict['file']['value']:
```

Comment:
@kuefmz I dont get this. what is this for. why is it called target file althoug it seems to be the group? variable???